db/db.js

```diff
@@ -21,19 +21,43 @@ exports.checkReviewExists = (review_id) => {
 
 exports.checkUsernameExists = (username) => {
   return db
-  .query(
-    `
+    .query(
+      `
     SELECT * FROM users
     WHERE username = $1
     `,
-    [username]
-  )
-  .then((result) => {
-    if (result.rows.length === 0) {
-      return Promise.reject({
-        status: 404,
-        msg: `${username} not found!`,
-      });
-    }
-  })
-}
+      [username]
+    )
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: `${username} not found!`,
+        });
+      }
+    });
+};
+
+exports.checkCategoryExists = (category) => {
+  console.log(category)
+  if (category === undefined) {
+    return Promise.resolve("potato")
+  }
+  console.log("BYEEE")
```

Comment:
44 - change so it is Promise.resolve() (we arent doing anything with the return)
46 - remove console.log